extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.json

```diff
@@ -0,0 +1,172 @@
+  ],
+  "description": "This Mixin contains the common Jounney Metadata information.",
+  "definitions": {
+    "journeyStepEventCommonFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
+          "title": "Journey Orchestration for creating user journey",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/journeyOrchestration/entrance":{
+              "title":"entrance",
+              "description":"Indicate if the user has entered the journey. If not present, assuming that the value is false.",
+              "type":"boolean"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/reentrance":{
+              "title":"reentrance",
+              "description":"Indicate if the user has reentered the journey with the same instance. If not present, assuming that the value is false.",
+              "type":"boolean"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/instanceEnded":{
+              "title":"instanceEnded",
+              "description":"Indicate if the instance is ended (sucessfully or not).",
+              "type":"boolean"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/eventId":{
+              "title":"eventId",
+              "description":"Event id in processing, for the step processing.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/nodeId":{
+              "title":"nodeId",
+              "description":"client node id.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/stepId":{
+              "title":"stepId",
+              "description":"Unique id of the step that is currently in processing.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/stepName":{
+              "title":"stepName",
+              "description":"Name of the step that is currently in processing.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/stepType":{
+              "title":"stepType",
+              "description":"Type of the step.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/stepStatus":{
+              "title":"stepStatus",
+              "description":"Status of the journey step, representing in which shape the step is, when its processing has been done.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/journeyId":{
+              "title":"journeyId",
+              "description":"ID of the journey.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/journeyVersionId":{
```

Comment:
Why do we need this property, if we have journeyVersionName

Reply:
This is the unique identifier for the journey version, but several Journey Version can have the same name.